tendrl/monitoring_integration/manager/__init__.py

```diff
@@ -0,0 +1,128 @@
+import __builtin__
+import traceback
+import sys
+import json
+
+import signal
+import maps
+import gevent
+import os
+
+from tendrl.monitoring_integration.grafana import utils
+from tendrl.monitoring_integration.grafana import exceptions
+from tendrl.monitoring_integration.grafana import dashboard
+from tendrl.monitoring_integration.grafana import datasource
+from tendrl.commons import manager as common_manager
+from tendrl import monitoring_integration
+from tendrl.monitoring_integration import sync
+from tendrl.commons import TendrlNS
+
+
+class MonitoringIntegrationManager(common_manager.Manager):
+
+    def __init__(self):
+
+        self._complete = gevent.event.Event()
+        super(
+            MonitoringIntegrationManager,
+            self
+        ).__init__(
+            sync.MonitoringIntegrationSdsSyncThread()
+        )
+
+    def start(self):
+        pass
+
+    def stop(self):
+        pass
+
+
+def upload_default_dashboards():
+
+        monitoring_integration_manager = MonitoringIntegrationManager()
+
+        # Creating Default Dashboards
+        dashboards = []
+        config = utils.get_conf("/etc/tendrl/monitoring-integration/" +
```

Comment:
you dont need to set the NS.config, you only need to define the config object [0]

Rest is done here https://github.com/Tendrl/commons/blob/master/tendrl/commons/__init__.py#L249

[0]: https://github.com/Tendrl/node-agent/blob/master/tendrl/node_agent/objects/config/__init__.py